src/components/Error/Error.tsx

```diff
@@ -73,7 +75,7 @@ export const Error = ({
         padding: theme.spacing(4, 0, 0, 4),
       })}
     >
-      <>
+      <Component>
```

Comment:
You can use the component prop on Box instead of adding another element to DOM

Reply:
`    <Box
      component={Component}
      sx={(theme) => ({
        overflow: 'auto',
        padding: theme.spacing(4, 0, 0, 4),
      })}
    >
      <>
        {/* Default status code to be helpful for the most common ones. The
      children can still override this by rendering <StatusCode /> themselves */}
        <StatusCode code={statusCode} />
        <Typography gutterBottom>Oops, Sorry.</Typography>
        {rendered}
        {!disableButtons && (
          <Grid
            container
            spacing={3}
            sx={{
              marginTop: 3,
            }}
          >
            <Grid item>
              <Button
                onClick={() => navigate(-1)}
                variant="contained"
                color="secondary"
              >
                Back
              </Button>
            </Grid>
            <Grid item>
              <ButtonLink to="/" variant="contained" color="secondary">
                Home
              </ButtonLink>
            </Grid>
          </Grid>
        )}
      </>
    </Box>`

removing the <Component> and just using the Box `component={Component}` prop throws the following error.
`This JSX tag's 'children' prop expects a single child of type 'ReactNode', but multiple children were provided.`

To get around this I can use a react Fragment as a parent container and it doesn't appear to add a new element to the DOM.  Will this work?